src/app/ReadClient.h

```diff
@@ -113,13 +113,14 @@ class ReadClient : public Messaging::ExchangeDelegate
          * receives an OnDone call to destroy the object.
          *
          * @param[in] apReadClient The read client object that initiated the read or subscribe transaction.
+         * @param[in] apVersion    The cluster version in report response.
```

Comment:
Which means what?  If it's null, what does that mean?  Needs to be clearly documented.

Reply:
updated, thanks, it would be nullptr when the value of EnableTagCompression in the Path field is true.